broker/api/v10.py

```diff
@@ -78,3 +78,59 @@ def submission_status(submission_id):
 @rest.get('/submissions/<submission_id>/log')
 def submission_log(submission_id):
     return u.render(api.submission_log(submission_id))
+
+""" Return the visualizer URL of a specific submission.
+
+    Normal response codes: 200
+    Error response codes: 400
+"""
+
+@rest.get('/submissions/<submission_id>/visualizer')
```

Comment:
This seems nice, I think it will be useful to provide a GET endpoint to retrieve the current activated cluster. 

Also, omit certificates and other sensitive information from the file.

Reply:
Modification made in commit 9c7e5e000ff2aa7bbe585c05ebc1c0a9755bb8e2